docs/playlist/soul.md

```diff
@@ -8,3 +8,4 @@ title: Soul
 | Playlist Name | Playlist Link | Platform | Playlist Author Name/Contributor Name | Note | Subgenre |
 | -------------- |-------------- | -------- | ------- | ------- | -------- |
 | Soul | [Spotify](https://open.spotify.com/playlist/6KaYLhkB6HXGhKHra1IClD?si=FRVxtkT7SK2gqnf9CVOnBg) | Spotify  | [Rommy Gustiawan](https://github.com/rgxcp) | - | - |
+| A Special One | [Spotify](https://open.spotify.com/playlist/7lqCQF4uBXoqgAMN9Qp3GD?si=881502329ba442d7) | Spotify  | [Romil Jain](https://github.com/romiljain23) | - | - |
```

Comment:
Please sort alphabetically, refer to this https://awesome-music-playlist.vercel.app/contribution.html#add-new-playlist